docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc

```diff
@@ -51,15 +51,29 @@ To take action to secure the assets, you can select the *Convert Unmanaged Asset
 
 Selecting the *Convert to Managed* button will onboard the assets to Prisma Cloud. Once the unmanaged assets convert to managed assets, Prisma Cloud will monitor these assets for vulnerabilities, misconfigurations, compliance issues, and so on.
 
-In addition to conversion options, the *Inspect Unmanaged Assets* column lists the scanned asset findings, enabling you to review the exposed assets. On selecting the links, a side panel appears, including a comprehensive list of assets, risk types, and associated cloud providers. Further exploration of each asset provides detailed information such as asset overview, exposed services, and identified findings.
+In addition to conversion options, the *Inspect Unmanaged Assets* column lists the scanned asset findings, enabling you to review the exposed assets. On selecting the links, a side panel appears, including a comprehensive list of assets, risk types, and associated cloud providers. Further exploration of each asset provides detailed information such as asset *Overview*, *Exposed Services*, identified *Findings*, and *Network Flow Log* visualizations.
```

Comment:
I would suggest a minor rewrite here:

In addition to conversion options, the *Inspect Unmanaged Assets* column lists scanned asset findings, enabling you to review exposed assets. Select a link to view the side panel. The panel includes a comprehensive list of assets, risk types, and associated cloud providers. Further exploration of each asset provides detailed information such as assets *Overview*, *Exposed Services*, identified *Findings*, and *Network Flow Log* visualizations.

Reply:
Done